api/pom.xml

```diff
@@ -380,7 +380,9 @@
                 </executions>
             </plugin>
 
+
             <!-- Create Javadoc for API jar -->
+            <!-- Result will be saved in ... ??? ... -->
```

Comment:
@arjantijms can you recheck? I expected it to end up in /target/site/apidoc but it was completely absent? As you've recently changed Javadoc generation I'm not sure if I have caught up it correctly.